### DISCUSSION TITLE
How to downmix to stereo

### DISCUSSION THREAD

**Discussion by dllu** [iina/iina#5253] [Q&A]
I only have 2 speakers plugged into my receiver. When I play videos with 5.1 audio tracks on Iina, dialog in the center channel is inaudible. In VLC, there is "headphones mode" which works perfectly. Is there something similar in Iina?

**Comment by low-batt** [comment]
For audio/video and subtitle rendering IINA is using a library from the [mpv](https://mpv.io/) project. Many of mpv's options that are not exposed directly in IINA's UI can still be used.

Checking the mpv manual the option [audio-channels](https://mpv.io/manual/stable/#options-audio-channels) may be what is needed:
> **--audio-channels=<auto-safe|auto|layouts>**
> …
>  - **--audio-channels=<stereo|mono>**
>        Force a downmix to stereo or mono. These are special-cases of the previous item. (See paragraphs below for implications.)

Normally I would test this before posting a comment, but I don't have the required audio setup. You will have to give it a try and report the results.

To enable this option follow these instructions:
- Start IINA
- Click on  `Settings…` under the `IINA` menu
- The settings panel appears
- On the left side of the panel click on `Advanced`
- Slide the `Enable advanced settings` toggle button to be on (blue)
- In the `Additional mpv options` section click on `+`
- A new entry appears in the table
- Double click on `name` and replace it with `audio-channels`
- Double click on `value` and replace it with `stereo`
- Confirm the panel looks like the screenshot below
- **Restart IINA** to activate the new settings
![audio-channels](https://github.com/user-attachments/assets/75fb3465-94f0-44ed-a80f-4d98f9f15905)
